core/src/test/java/io/confluent/rest/ErrorHandlerIntegrationTest.java

```diff
@@ -34,9 +37,14 @@ public void tearDown() throws Exception {
   }
 
   @Test
-  public void unhandledServerExceptionDisplaysStackTrace() throws Exception {
-    TestApplication application = new TestApplication(new RestConfig(RestConfig.baseConfigDef()),
-        false);
+  public void unhandledServerExceptionDisplaysStackTraceForInvalidAuthentication()
+      throws Exception {
+
+    Properties props = new Properties();
+    props.setProperty(RestConfig.ERROR_HANDLING_ENABLED_CONFIG, "false");
```

Comment:
I'm wondering if this is the right name for the property.

What the config really does is SUPPRESS_STACK_TRACE_IN_RESPONSE so something similar to that might be more appropriate. What do you think?